### PR TITLE
styling & templating 

### PR DESCRIPTION
**Pull request by lschatzkin**
blog, materials, lessons

### CODE REVIEW

File: app/assets/stylesheets/application/components/footer.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 footer {
+  display: none; //small screens
```
Comment (wioux) [inline app/assets/stylesheets/application/components/footer.scss]: Why don't we do footers on mobile?

Comment (lschatzkin) [reply app/assets/stylesheets/application/components/footer.scss]: In the design, the nav is going to be fixed at the bottom of the screen.

**Comment by wioux** [comment]
Looking good.

Can you clarify what the application/components directory is all about? What goes in there vs. directly under application/?

**Comment by lschatzkin** [comment]
The file organization has gotten wonky.  What do you think of having sections like lessons, materials, topics under application and items like cards, tags, etc. under components?

**Comment by wioux** [comment]
I like that idea :+1:

**Comment by lschatzkin** [comment]
Cool, I'll do that now and push the changes here.

**Comment by wioux** [comment]
Great. Thanks!

**Pull request merged**